docs/release-notes.txt

```diff
@@ -8,6 +8,52 @@
  Version 0.14
 **************
 
+Version 0.14.6
+==============
+
+**Release Date:** April 1, 2021
+
+**New Features**
+
+-  API: Add a new endpoint to delete experiments. This endpoint is only
+   enabled for admin users and deletes all resources associated with an
+   experiment. This includes checkpoint storage, Tensorboards, trial
+   logs from all backends and metadata such as history and metrics,
+   stored in PostgreSQL.
+
+-  API: Add a new endpoint to fetch aggregated historical resource
+   allocation information.
+
+-  CLI: Add new commands ``det resources raw`` and ``det resources
+   aggregated`` to access resource allocation information.
+
+-  PyTorch Lightning: Add an adapter to support ``LightningModule`` from
```

Comment:
blocking: We typically either say "See `<page-title-as-link>` for more information", or "For more information, reference our documentation `<link>`here`</link>`." typically we don't combine the word "here" with the entire sentence being linkified.

Reply:
oh no i misclicked. this is supposed to be for line 32